src/components/ftbot/PairlistConfigItem.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <b-card no-body class="mb-2">
+    <template #header>
+      <div class="d-flex text-start align-items-center">
+        <div class="d-flex flex-grow-1 align-items-center">
+          <i-mdi-reorder-horizontal
+            role="button"
+            class="handle me-2 fs-4 flex-shrink-0"
+            width="24"
+            height="24"
+          />
+          <div
+            role="button"
+            class="d-flex flex-grow-1 align-items-start flex-column"
+            @click="toggleVisible"
+          >
+            <span class="fw-bold">{{ pairlist.name }}</span>
+            <span class="text-small">{{ pairlist.description }}</span>
+          </div>
+        </div>
+        <i-mdi-close
+          role="button"
+          width="24"
+          height="24"
+          class="mx-2"
+          @click="pairlistStore.removeFromConfig(index)"
+        />
+        <i-mdi-chevron-down
+          v-if="!visible"
+          :class="hasParameters && !visible ? 'visible' : 'invisible'"
+          role="button"
+          class="fs-4"
+          @click="toggleVisible"
+        />
+        <i-mdi-chevron-up
+          v-if="visible"
+          :class="hasParameters && visible ? 'visible' : 'invisible'"
+          role="button"
+          class="fs-4"
+          @click="toggleVisible"
+        />
+      </div>
+    </template>
+    <b-collapse v-model="visible">
+      <b-card-body>
+        <PairlistConfigParameter
+          v-for="(parameter, key) in pairlist.params"
+          :key="key"
+          v-model="pairlist.params[key].value"
+          :param="parameter"
+        />
+      </b-card-body>
+    </b-collapse>
+  </b-card>
+</template>
+
+<script setup lang="ts">
+import PairlistConfigParameter from '@/components/ftbot/PairlistConfigParameter.vue';
+import { usePairlistConfigStore } from '@/stores/pairlistConfig';
+import { Pairlist } from '@/types';
+import { computed, ref } from 'vue';
+
+const pairlistStore = usePairlistConfigStore();
+
+defineProps<{
+  index: number;
+}>();
+
+const visible = ref(false);
```

Comment:
this causes odd behavior - if i multiple parameters extended and add another pairlist, all pairlists will collapse - i assume because the state is local, and the list will change, causing a re-render of all items.
we might need to move this ot the store - effectively having it a part of "Pairlist" - even though it'll never leave the UI.